src/components/vm/overview/bootOrder.tsx

```diff
@@ -129,9 +149,9 @@ const DeviceRow = ({ idPrefix, device, index, onToggle, upDisabled, downDisabled
             switch (device.device.type) {
             case "usb": {
                 addOptional(additionalInfo, device.device.type, _("Type"));
-                addOptional(additionalInfo, nodeDev.capability.vendor._value, _("Vendor"));
-                addOptional(additionalInfo, nodeDev.capability.product._value, _("Product"));
-                if (nodeDevs.length > 1) {
+                addOptional(additionalInfo, nodeDev.capability.vendor?._value, _("Vendor"));
+                addOptional(additionalInfo, nodeDev.capability.product?._value, _("Product"));
+                if (nodeDevs.length > 1 || typeof nodeDev.capability.bus != "string") {
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2185-af23a7d6-20250716-112918-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)